src/dotnet-install.sh

```diff
@@ -869,7 +869,7 @@ extract_dotnet_package() {
 }
 
 # args:
-# remote_path - $1
+# remote_path - $1 - Note, credentials will not be applied.
```

Comment:
As I understand, we are still passing the credentials to aka.ms links in bash script like: `aka.ms/dotnet/internal/daily/some_archive.zip?credentials`. 
But in powershell script, aka.ms doesn't get the credentials. Is it intended that they are different?

Reply:
The code structure related to downloading differs for sh and ps1 scripts. So, removal of the credentials from aka.ms links was done differently.
So, no, we are not adding credentials to aka.ms link, I removed that from this function, you can see it below. The comment actually indicates this fact. We no more apply credentials in this function, which is used only for resolution of aka.ms links into a direct ones.

Reply:
$remote_path_with_credential -> $remote_path is the related change, removing the credentials from aka.ms links. It is done in `get_http_header_curl` and `get_http_header_wget`. This function is just a generic one, it allows us not to care about what we have on the machine for downloading, wget or curl.